### ISSUE TITLE
Describe tunnel aspect properties

### ISSUE THREAD

**Issue by md5i** [CyclopsMC/IntegratedDynamics#786]
#### Issue type:

- :heavy_plus_sign: Feature request

____

#### Short description:

Currently, tunnel aspect properties are not described in the manual or via tool tips.  As many of the properties' uses are not immediately discernible based on their name alone, some description would be appreciated.

Examples:  

- Dispense (What does changing this boolean do?)
- Exact Amount (Where does the amount come from?  What would this mean for, say, the list version of Place Item Entities?)
- Blacklist

**Comment by rubensworks** [comment]
I think some of those have a description that is shown when hovering over their name in the aspect settings gui. But I guess some improvements may be possible there indeed.

**Comment by md5i** [comment]
> I think some of those have a description that is shown when hovering over their name in the aspect settings gui. But I guess some improvements may be possible there indeed.

Indeed, I wasn't able to find those tooltips without your description.  Though I still wonder things like, Exact Amount (which does have a tool tip) comes from. (How is the amount specified?  In an itemstack somehow?)  And Dispense has no description.

At the very least, it would be nice if these were described in the manual.  Thanks for considering this.
